.github/workflows/shared_meterpreter_acceptance.yml

```diff
@@ -251,6 +251,35 @@ jobs:
           path: metasploit-payloads
           ref: ${{ inputs.metasploit_payloads_commit }}
 
+      - name: Get metasploit-payloads version
+        if: ${{ inputs.build_metasploit_payloads }}
+        shell: bash
+        run: |
+          echo "METASPLOIT_PAYLOADS_VERSION=$(ruby -ne 'puts $& if /[0-9].[0-9].[0-9]*/' gem/lib/metasploit-payloads/version.rb)" | tee -a $GITHUB_ENV
+        working-directory: metasploit-payloads
+
+      - name: Build metasploit-payloads gem
+        if: ${{ inputs.build_metasploit_payloads }}
+        run: |
+          gem build ./gem/metasploit-payloads.gemspec
+        working-directory: metasploit-payloads
+
+      - name: Copy metasploit-payloads gem into metasploit-framework
+        if: ${{ inputs.build_metasploit_payloads }}
+        shell: bash
```

Comment:
I don't think explicitly asking for bash is required here.

Reply:
Hmm, I added bash here so that I could use this single command for all OS's. Without bash I had to have one for Windows using the `copy` command and then another for the rest of the OS's. At least from what I tested that was the case.